files/build_templates/docker_image_ctl.j2

```diff
@@ -234,6 +245,19 @@ start() {
                         for elem in data['INCLUDES'] if 'namespace' in elem])); f.close()" $SONIC_DB_GLOBAL_JSON`
     fi
 
+    global_server="redis_global.server"
+    global_db_address="127.100.0.1"
+    redis_global_server="$( grep $global_server /etc/hosts )"
+    if [ -n "$redis_global_server" ]; then
+        global_db_address="$( grep "redis_global.server" /etc/hosts | cut -f 1 -d ' ' )"
+    fi
+    global_db_opt="--add-host=$global_server:$global_db_address"
+    start_globaldb=$(sonic-cfggen -G -v GLOBALDB_CONFIG.start_global_db)
+    globaldb_address=$(sonic-cfggen -G -v GLOBALDB_CONFIG.global_db_address)
```

Comment:
@tahmed-dev , what is the way to optimize this sonic-cfggen, we'd like to reduce the sonic-cfggen call in the startup.

Reply:
ACK. @BrynXu can you please json template file with `start_global_db` and `global_db_address`? Here is an [example](https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-orchagent/swss_vars.j2)

Reply:
@tahmed-dev I will follow your counter example of parsing config file [example](https://github.com/Azure/sonic-buildimage/blob/master/files/image_config/config-setup/config-setup#L212) to parse globaldb.conf directly in code instead of calling sonic-cfggen.